src/main/java/com/milliondollawinners/Route.java

```diff
@@ -0,0 +1,58 @@
+package com.milliondollawinners;
+
+public class Route {
+    private Location startLoc;
+    private Location endLoc;
+    //private Time startTime;
+    //Distance in miles
+    private double distance;
+    //Elevation gain in feet
+    private int elevationGain;
+
```

Comment:
I wonder if we should add ArrayLists for lat/lon waypoints to describe the route